sudoku_board.cc

```diff
@@ -1,123 +1,181 @@
 
 #include "sudoku_board.h"
 
-// ------------------------------- public methods ---------------------------------
-
-SudokuBoard::SudokuBoard(int size){
-    m_size = size;
-    m_board_values = std::vector(size, std::vector<int>(size, EMPTY_CELL_SIGN));
-    m_possible_options = std::vector(size, std::vector(size, std::set<int>(POSSIBLE_CELL_VALUES)));
+// constant
+const std::set<int> SudokuBoard::POSSIBLE_CELL_VALUES = {1, 2, 3, 4, 5,
+                                                         6, 7, 8, 9};
+const int SudokuBoard::EMPTY_CELL_SIGN = 0;
+const size_t SudokuBoard::CUBE_SIZE = 3;
```

Comment:
Think about how to make the other constants be dependent on `CUBE_SIZE` so that when you change this value to 4, everything will work but with a bigger board